README.rst

```diff
@@ -24,5 +24,5 @@ of a component, check the `Home Assistant help section <https://home-assistant.i
    :target: https://www.home-assistant.io/join-chat/
 .. |screenshot-states| image:: https://raw.githubusercontent.com/home-assistant/core/master/docs/screenshots.png
    :target: https://demo.home-assistant.io
-.. |screenshot-integrations| image:: https://raw.githubusercontent.com/home-assistant/core/dev/docs/screenshot-integrations.png
+.. |screenshot-integrations| image:: https://raw.githubusercontent.com/home-assistant/core/master/docs/screenshot-integrations.png
```

Comment:
This will not work after the next version either. The folder `docs` has been removed with https://github.com/home-assistant/core/pull/102274.

So we have to place or get this image and the above screenshot somewhere else. 
@frenck do we have them somewhere else or should they be completely removed from the README?

Reply:
I think we can include a generic screenshot of the system for the first one.

For the featured integrations, I would suggest on remove it from the readme completely (or link to them). The screenshots need updating all the time, and this is not the place for them IMHO.

../Frenck